rllib/algorithms/algorithm_config.py

```diff
@@ -3194,6 +3197,14 @@ def experimental(
         """Sets the config's experimental settings.
 
         Args:
+            _torch_grad_scaler_class: Class to use for torch loss scaling (and gradient
+                unscaling). The class must implement the following methods to be
+                compatible with a `TorchLearner`. These methods/APIs match exactly the
+                those of torch's own `torch.amp.GradScaler`:
+                `scale([loss])` to scale the loss.
+                `get_scale()` to get the current scale value.
+                `step([optimizer])` to unscale the grads and step the given optimizer.
+                `update()` to update the scaler after an optimizer step.
```

Comment:
The description is not clear enough imo. For what is it used?

Reply:
enhanced and added a link to the torch docs